resources/views/layout.blade.php

```diff
@@ -1,32 +0,0 @@
-<Html>
-
-<head>
-    <title>
-        @yield('title')
-    </title>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
-</head>
-
-<body>
-    <div class="container">
-        <div class="row mt-5 justify-content-center">
-            <div class="col-md-8">
-                <div class="card">
```

Comment:
Tip: for the future when you wanted to move or rename an file, just try to use `git mv` . it will produce better **diff**